src/v2/controllers/FormController.js

```diff
@@ -156,6 +161,13 @@ export default Controller.extend({
         .then(this.handleIdxSuccess.bind(this))
         .catch(error => {
           this.showFormErrors(this.formView.model, error);
+        })
+        .then(() => {
+          if (actionPath === 'cancel' && useInteractionCodeFlow) {
```

Comment:
does this need to be taken care in `handleSaveForm` as well ?

Reply:
In my understanding, no. As we need to restart login flow only on cancel, which is triggered from here
https://github.com/okta/okta-signin-widget/blob/194575de499cb4ce91453362a517623256dbab87/src/v2/view-builder/components/Link.js#L57
and handled in `handleInvokeAction`

Reply:
Why only on `cancel` ? can there be token/code expiration ? what if enduser do a hard refresh on this page when token is expired ?

Reply:
If token is expired, user will see screenshot 1. Click on 'Back to sign-in' will restart login flow correctly.
<img width="419" alt="Screenshot 2021-07-12 at 22 19 17" src="https://user-images.githubusercontent.com/72614880/125344161-a1bf5000-e35f-11eb-9ca7-0c42aa463ce2.png">

If user will do a hard refresh with expired token, user will see screenshot 2. Click on  'Back to sign-in' will do nothing as it's just goto link. I fixed this in last commit by setting `actionPath: cancel` and removing `'href': settings?.get('baseUrl'),` 
<img width="425" alt="Screenshot 2021-07-12 at 22 19 46" src="https://user-images.githubusercontent.com/72614880/125344171-a4ba4080-e35f-11eb-9b1e-c097dfd770cb.png">

WDYT?

Reply:
Will move fix for expired session to another PR

Reply:
Let's setup a meeting to discuss this change.. I am not comfortable with this one-off chaining here for interaction flow.. why it needs a special treatment.

Reply:
@denysoblohin-okta will create jira to remove exclusive handling of cancel action for `interaction code flow` for devex team to resolve